src/main/java/helloworld/s3_lambda_sqs/S3Bucket.java

```diff
@@ -0,0 +1,75 @@
+package helloworld.s3_lambda_sqs;
+
+import com.amazonaws.AmazonServiceException;
+import com.amazonaws.regions.Regions;
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.AmazonS3ClientBuilder;
+import com.amazonaws.services.s3.model.AmazonS3Exception;
+import com.amazonaws.services.s3.model.Bucket;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.util.List;
+import java.util.Objects;
+
+import static helloworld.constant.Constants.BUCKET_FOUND;
+import static helloworld.constant.Constants.BUCKET_NAME;
+import static helloworld.constant.Constants.CURRENT_BUCKET;
+import static helloworld.constant.Constants.FILE_PATH;
+import static helloworld.constant.Constants.KEY_NAME;
+import static helloworld.constant.Constants.SUCCESS_WRITE_TO_S3;
+
+public class S3Bucket {
+    Gson gson = new GsonBuilder().setPrettyPrinting().create();
+    private static final Logger logger = LoggerFactory.getLogger(S3Bucket.class);
+
+    public static void main(String[] args) {
+        final AmazonS3 s3 = AmazonS3ClientBuilder.standard().withRegion(Regions.EU_WEST_1).build();
+
+        Bucket bucket = createBucket();
+        Objects.requireNonNull(bucket);
+
+        try {
+            s3.putObject(BUCKET_NAME, KEY_NAME, new File(FILE_PATH));
+        } catch (AmazonServiceException e) {
+            logger.error(e.getErrorMessage());
+            System.exit(1);
```

Comment:
`System.exist(1)` should be used in contexts like described here: https://stackoverflow.com/a/3716015
In the case of Lambdas, we can just throw the exception up, and the AWS Lamda engine takes care of the rests.

Also another important point to consider is that, if System.exit would kill our program as it's intended with System.exit, we would loose the context sharing of our Lambda, leading to more often Cold Starts. Some interesting material about it here: https://aws.amazon.com/blogs/compute/new-for-aws-lambda-predictable-start-up-times-with-provisioned-concurrency/